fusor-ember-cli/.gitignore

```diff
@@ -5,8 +5,10 @@
 /tmp
 
 # dependencies
-/node_modules
-/bower_components
+/node_modules*
+/bower_components*
+.nvmrc
+use-ember-cli*.sh
```

Comment:
this is for my local scripts to remove and add symlinks that I'm not sure is needed in the repo. If people want them there, I'm happy to add them.

It's just two files. Here's `use-ember-cli-v1.13.sh`
```
 #!/bin/bash

npm uninstall -g ember-cli
npm install -g ember-cli@1.13.12

npm cache clean
bower cache clean

rm node_modules
ln -s node_modules_1_13 node_modules

rm bower_components
ln -s bower_components_1_13 bower_components

npm install
bower install
```